skillful_network_client/src/app/home/news-feed/add-publication/add-publication.component.html

```diff
@@ -0,0 +1,17 @@
+
+<div class='card mat-elevation-z4' id="publication-card">
```

Comment:
Could we reuse the abstract component `card.component` in the `shared/component/` folder? In fact, you would inherit from the style and avoid duplication of Code.